libvips/foreign/svgload.c

```diff
@@ -315,6 +325,9 @@ vips_foreign_load_svg_file_header( VipsForeignLoad *load )
 
 static const char *vips_foreign_svg_suffs[] = {
 	".svg",
+#if LIBRSVG_CHECK_FEATURE(SVGZ)
```

Comment:
I think this should be `LIBRSVG_CHECK_FEATURE(SVGZ) && defined(HAVE_ZIP)` to match the other ifs. I'll make a #define for this test to be safe.